autoware_launch/config/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/obstacle_avoidance_planner.param.yaml

```
@@ -52,6 +51,8 @@
         lat_error_weight: 0.001
 
       qp:
+        solver: "proxqp"
```

How about adding the possible values as comment ?
```suggestion
        solver: "proxqp"  # QP solver used. Can be "osqp" or "proxqp".
```
Same for the MPT parameter.

---

Sounds much better. Thank you.